plugin.go

```diff
@@ -34,13 +34,17 @@ type AuthProvider struct {
 	Context         string             `json:"context,omitempty"`
 	Master          bool               `json:"master,omitempty"`
 	TokenName       string             `json:"token_name,omitempty"`
-	TokenSecret     string             `json:"token_secret,omitempty"`
+	TokenHSSecret   string             `json:"token_secret,omitempty"`
 	TokenIssuer     string             `json:"token_issuer,omitempty"`
 	AuthURLPath     string             `json:"auth_url_path,omitempty"`
 	AccessList      []*AccessListEntry `json:"access_list,omitempty"`
 	CommonTokenParameters
-	logger         *zap.Logger     `json:"-"`
 	TokenValidator *TokenValidator `json:"-"`
+
+	TokenRS
+
+	logger    *zap.Logger
+	tokenKeys map[string]interface{} // the value must be a *rsa.PrivateKey or *rsa.PublicKey
```

Comment:
@njones, how about moving this map to `RsaSignMethodConfig`?

Reply:
For this I would like to keep here only because I was thinking ahead of adding the `ES` signing methods next, so this can hold `RS`, `ES` (and `PS`) key pairs in the near future, and this would be the place to store any key pairs.